src/SamplesApp/SamplesApp.Shared/Samples/UnitTests/UnitTestsPage.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<Page
+    x:Class="SamplesApp.Samples.UnitTests.UnitTestsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:SamplesApp.Samples.UnitTests"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:ctl="using:Uno.UI.Samples.Tests"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
+		<ctl:UnitTestsControl />
```

Comment:
Suggestion : xmlns should be "controls" to follow https://nventive.visualstudio.com/Umbrella/_wiki/wikis/Umbrella.wiki?wikiVersion=GBwikiMaster&pagePath=%2FProgramming%20guidelines%2FGuidance%2FDevelopment%20standards%2FXAML&anchor=xaml-namespaces-(xmlns)